packages/instrumenter/src/parsers/svelte-parser.ts

```diff
@@ -0,0 +1,100 @@
+import { BaseNode, Program } from 'estree';
```

Comment:
Nit: These are type imports only. Marking them with `import type` will make sure no value import remeans after TypeScript compilation.

```suggestion
import type { BaseNode, Program } from 'estree';
```